src/breadcrumb/breadcrumb.scss

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-breadcrumb {
+  @include iui-reset;
+  @include iui-font-family;
+  display: flex;
+  align-items: center;
+
+  > .iui-button {
+    margin-right: $iui-s;
+  }
+  
+  ol {
+    @include iui-reset;
+    list-style-type: none;
+    display: flex;
+    align-items: center;
+    user-select: none;
+
+    li {
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      max-width: $iui-3xl * 2;
+
+      a:not([aria-current]) {
+        @include iui-anchor;
+      }
+
+      [aria-current] {
```

Comment:
This is bad for many reasons, but also it is buggy because it would match `aria-current=false` too.

Reply:
I did some quick Googling but didn't find an answer… any idea if you can do this in SCSS?
```
[aria-current=true] { … }
```

Anyways, I added an `.iui-current` class for now.

Reply:
you can do that but we don't know if the value will be `true` or `location` or `page` (all of which are valid to describe current breadcrumb)